### PR TITLE
updated doc with troubleshooting section for console logout issue

### PR DESCRIPTION
**Pull request by dlaloue-vmware**
### Description of the change

Added troubleshooting section for the console logout issue reported in https://github.com/kubeapps/kubeapps/issues/2686
Updated a few sample sections as well to include the new option.

### Benefits

Users may encounter the issue where they are unexpectedly logged out of the console. The new section helps identify the issue and provide a way to configure Kubeapps to prevent/fix the issue.

### CODE REVIEW

File: docs/user/using-an-OIDC-provider.md
Comment (absoludity) [inline docs/user/using-an-OIDC-provider.md:368]: Isn't this statement going to be incorrect once your chart change lands (as it's setting a sensible default, which ever way it does so)?

File: docs/user/using-an-OIDC-provider.md
Comment (absoludity) [inline docs/user/using-an-OIDC-provider.md:370]: Similar to my comment on the chart, I think this would be a better UX to be able to just set a chart value directly. Guess we'll want to wait until the chart change is landed before updating this.

File: docs/user/using-an-OIDC-provider.md
Comment (dlaloue-vmware) [reply docs/user/using-an-OIDC-provider.md:368]: this will be obsolete for new installation / upgrades once we have the new chart.
but this is an existing problem for users, so i think it is useful to have a troubleshooting section for it.
I agree that i will have to reword it once we have the change in the chart.

File: docs/user/using-an-OIDC-provider.md
Comment (dlaloue-vmware) [reply docs/user/using-an-OIDC-provider.md:370]: yes, this was worded with assumption the new chart is not yet available.
if we wait for the chart, i will reword accordingly (reason why i marked the PR as draft :-)

File: docs/user/using-an-OIDC-provider.md
Comment (absoludity) [inline docs/user/using-an-OIDC-provider.md:368]:
```suggestion
Prior to Kubeapps 7.0.5, the auth proxy configuration did not include a default `--cookie-refresh` value to refresh the access/openid token and so the console will logout once the token expires. In the case of Keycloak for example, this can happen quickly as the default access token expiration is 5m.
```

File: docs/user/using-an-OIDC-provider.md
Comment (absoludity) [inline docs/user/using-an-OIDC-provider.md:371]:
```suggestion
 - upgrade Kubeapps to version 7.0.5+ which sets a default of `--cookie-refresh=2m` and exposes the value in the chart values as `authProxy.cookieRefresh`.
```

File: docs/user/using-an-OIDC-provider.md
Comment (absoludity) [inline docs/user/using-an-OIDC-provider.md:376]: Not sure how relevant this is given that we recommend the `oidc` provider which is supported. But yep, people aren't restricted to that (eg. I've used the google provider in certain cases). Perhaps:
```suggestion
**Note**: If you have configured a provider other than `oidc` for oauth2-proxy, the issue may still occur even after upgrading or updating Kubeapps as OAuth2 Proxy does not support cookie-refresh for all providers. See [OAuth2 Proxy Configuration Overview](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview/#footnote1) for the list of supported providers.
```

**Review by absoludity — APPROVED** [review]
A couple of suggestions but feel free to ignore them if you think otherwise.

File: docs/user/using-an-OIDC-provider.md
Comment (antgamdia) [inline docs/user/using-an-OIDC-provider.md:368]:
```suggestion
Prior to the Kubeapps chart version 7.1.0, the auth proxy configuration did not include a default `--cookie-refresh` value to refresh the access/openid token and so the console will logout once the token expires. In the case of Keycloak for example, this can happen quickly as the default access token expiration is 5m.
```

File: docs/user/using-an-OIDC-provider.md
Comment (antgamdia) [inline docs/user/using-an-OIDC-provider.md:371]:
```suggestion
 - upgrade the Kubeapps chart to version 7.1.0+ which sets a default of `--cookie-refresh=2m` and exposes the value in the chart values as `authProxy.cookieRefresh`.
```

**Review by antgamdia — APPROVED** [review]
Great, thank you for performing these changes.
I've also noticed a couple of (already existing minor typos),  if you want, check them; but feel free to ignore them as they are not from this PR.

- Remove duplicated "accept" in: `accept accept access_tokens from their identity provider as bearer tokens (see GKE below).`
- Replace "prodcution" by "production" in `the prodcution VMware cloud services issuer`

**Comment by absoludity** [comment]
You'll need to switch this from a draft PR to be able to land it, I think, if you think it's ready. It's got +1s.

**Comment by antgamdia** [comment]
@dlaloue-vmware, feel free to click "squash and merge" whenever you want. Looking forward to have the docs up to date with these changes  :)

**Pull request merged**
